src/routes/ConsortiumManager/settings/users/general/PermissionSets/PermissionSets.js

```diff
@@ -125,6 +225,18 @@ export const PermissionSets = (props) => {
     >
       <Switch>
         <Route exact path={PERMISSION_SET_ROUTES.COMPARE} component={PermissionSetsCompare} />
+        <Route exact path={PERMISSION_SET_ROUTES.CREATE}>
```

Comment:
it should be `<Route exact path={PERMISSION_SET_ROUTES.CREATE} component={PermissionSetsCreate}>`